utils/contexts/writingPanel/WritingPanelProvider.tsx

```diff
@@ -23,16 +23,24 @@ export const WritingPanelProvider = ({ children }: { children: ReactNode }) => {
   );
   const action: WritingPanelAction = {
     setFormData: (data) => {
+      // encoding content
       const processedData: WritingPanelFormProps = {
         ...data,
         content: encodeURIComponent(data.content),
       };
+      // set state
       dispatch({ type: "SET_FORM_DATA", payload: { data: processedData } });
-      // console.log(processedData);
-      storageSave(KEY_ARTICLE_DRAFT, JSON.stringify(data));
+      // save to local
+      storageSave(
```

Comment:
## Main culprit

yep this is the main culprit, because if we have a `FileList` type with a child (applies with multiple children to i think...) and we stringify it, it will just be an array with 1 child that is an exact empty object, like this : `[{}]`

and then if we try to get that value, in this case `formData.thumbnail.[0]` that won't work cuz all we get is just an empty object and `URI.createObjectURL` will be mad if we feed them that.

Great fix, thanks

Reply:
Yea, i just learned that today too.. TIL!